relay-server/src/services/projects/cache/state.rs

```diff
@@ -271,6 +272,41 @@ impl ProjectRef<'_> {
 
     fn complete_fetch(&mut self, fetch: CompletedFetch, config: &Config) -> Option<ExpiryTime> {
         let now = Instant::now();
+
+        // Note: currently the latency also requires the project to be not disabled,
+        // because we do not store the last change timestamp for disabled projects,
+        // this limitation could be lifted.
```

Comment:
This comment is a bit confusing. Does "because we do not store the last change timestamp for disabled projects" refer to the preceding clause ("currently the latency also requires the project to be not disabled") or the following one ("this limitation could be lifted")?

Reply:
Moved the comment to a better stop with better wording!